src/DynamoCore/Graph/Workspaces/WorkspaceModel.cs

```diff
@@ -1320,6 +1321,16 @@ internal void IncrementPasteOffset()
         #endregion
 
         #region private/internal methods
+        /// <summary>
+        /// Returns true if the graph currently contains dummy node which point to XML content.
+        /// These nodes cannot be serialized to json correctly.
+        /// </summary>
+        /// <returns></returns>
+        internal bool containsXmlDummyNodes()
+        {
+            //if the workspace being added has dummy nodes which contain XML, log a notification.
```

Comment:
Should we move this comment to the place we actually log a notification not here?